pytest_catchlog/plugin.py

```diff
@@ -50,12 +56,60 @@ def pytest_addoption(parser):
         dest='log_date_format', default=DEFAULT_LOG_DATE_FORMAT,
         help='log date format as used by the logging module.'
     )
+    add_option_ini(
+        parser,
+        '--log-cli-level',
+        dest='log_cli_level', default=None,
+        help='cli logging level.'
+    )
+    add_option_ini(
+        parser,
+        '--log-cli-format',
+        dest='log_cli_format', default=None,
+        help='log format as used by the logging module.'
+    )
+    add_option_ini(
+        parser,
+        '--log-cli-date-format',
+        dest='log_cli_date_format', default=None,
+        help='log date format as used by the logging module.'
+    )
+
+
+def get_actual_log_level(config, setting_name):
+    '''
+    Return the actual logging level
+    '''
```

Comment:
If we want to follow pep257, that should be `"""Return the actual logging level."""` - all in one line and with a period :wink: